### DISCUSSION TITLE
[1260810]东方华彩乱战2 ~ Touhou Blooming Chaos 2 本体+dlc

### DISCUSSION THREAD

**Discussion by fengse2020** [pjy612/SteamManifestCache#55] [Share]
appid:1260810
code:FB52C5230A918469
email:2646331095@qq.com
[东方华彩乱战2 ~ Touhou Blooming Chaos 2.zip](https://github.com/pjy612/SteamManifestCache/files/13692660/2.Touhou.Blooming.Chaos.2.zip)

**Comment by pjy612** [comment]
查收

**Comment by mianbaoyu01** [comment]
怎么哪里都有越共呀（恼
